runtime/compiler/CMakeLists.txt

```diff
@@ -80,10 +80,10 @@ add_custom_command(
 )
 add_custom_target(j9jit_tracegen DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/env/ut_j9jit.h)
 
-# JITSERVER_SUPPORT and protobuf
-if(JITSERVER_SUPPORT)
+# J9VM_OPT_JITSERVER and protobuf
+if(J9VM_OPT_JITSERVER)
 	message(STATUS "JITServer is supported")
-	add_definitions(-DJITSERVER_SUPPORT)
+	add_definitions(-DJ9VM_OPT_JITSERVER)
```

Comment:
I am not sure if defines in `runtime/CMakeLists.txt` propagate to JIT `CMakeLists.txt`. If so, then yes it should be removed.
@keithc-ca Can you please confirm it.

Reply:
@dnakamura might know off the top of his head, but please try without what I hope is redundant.

Reply:
Instead of using an `add_definition` the proper way would be to add `#cmakedefine J9VM_OPT_JITSERVER` to `runtime/include/j9cfg.h.in`

Reply:
I think that will play better with DDR should `J9VM_OPT_JITSERVER` ever need to be used there.

Reply:
@dnakamura how would cmake know whether to define or undef `J9VM_OPT_JITSERVER` in the generated `j9cfg.h` file?

Reply:
Would I need to add the following towards the end of `cmake/caches/common.cmake`:
```
if (J9VM_OPT_JITSERVER)
    set(J9VM_OPT_JITSERVER ON CACHE BOOL "")
endif()
```
and an entry in `cmake/options.cmake`:
```
option(J9VM_OPT_JITSERVER, "some text")
```

Reply:
In some cmake file, we want
```
set(J9VM_OPT_JITSERVER OFF CACHE BOOL "Enable JIT Server")
```
This tells cmake about the flag so it know what to do with `#cmakedefine J9VM_OPT_JITSERVER`.

Notice the default is `OFF`; it can be modified by a command-line option or via logic in cmake files.

Reply:
I think I am getting confused with env variable `J9VM_OPT_JITSERVER` and cmake variable `J9VM_OPT_JITSERVER`.
Lets say I have added `set(J9VM_OPT_JITSERVER OFF CACHE BOOL "Enable JIT Server")` to `cmake/caches/common.cmake`.
Now if env variable `J9VM_OPT_JITSERVER` is defined, is that sufficient for cmake to convert `#cmakedefine J9VM_OPT_JITSERVER` to `#define J9VM_OPT_JITSERVER` in the header file?

Reply:
In the cmake case, it is not an environment variable, but this command-line option that affects that flag:
```
CMAKE_ARGS += -DJ9VM_OPT_JITSERVER=ON
```